pages/index.js

```diff
@@ -5,47 +5,48 @@ import TodoList from "../components/todo-list";
 
 import styled from "styled-components";
 
-let taskFlag = 0;
-let dateFlag = 0;
 export default function Home() {
   const [date, setDate] = useState();
   const [task, setTask] = useState();
   const [todos, setTodoList] = useState([]);
 
   const onChangeTask = (e) => {
     setTask(e.target.value);
-    taskFlag++;
   };
   const onChangeDate = (e) => {
     setDate(e.target.value);
-    dateFlag++;
   };
 
   const stateReset = (e) => {
-    dateFlag = 0;
-    taskFlag = 0;
     setDate("");
     setTask("");
   };
-  const addTodo = (e) => {
-    if (dateFlag === 0 || taskFlag === 0) {
+
+  const validateTodo = () => {
+    let isValid = false;
+    let isDateValid = date.length === 8 && date > 20200101;
```

Comment:
```suggestion
    const isDateValid = date.length > 8 && date > 20200101;
```
1. 오타가 있네요 😅
2. 값에 변화가 없어서 const를 써도될 것 같아요~